about_sigils.exs

```diff
@@ -0,0 +1,34 @@
+defmodule AboutSigils do
+  use Koans
+
+  think "The ~s sigil is a different way of expressing string literals" do
+    assert_? "This is a string" == ~s{This is a string}
+  end
+
+  think "Sigils can come in handy" do
+    assert "\"Hello, I am a string\" - \"Hey, me too\"" == __?
```

Comment:
This one ended up being _super_ confusing to me at first. I was stuck thinking "oh huh you can subtract sigils"? I think it would be improved by editing the description and perhaps the test value.

```elixir
think "Sigils are useful to avoid escaping quotes in strings" do
  assert "\"Welcome to the jungle\", they said." == __?
end
```